source/uwp/Visualizer/ViewModel/GenericDocumentViewModel.cs

```diff
@@ -117,15 +120,58 @@ protected async Task LoadFromFileAsync(IStorageFile file, string token, bool ass
             }
         }
 
+        private async Task CreateMessageDialog(string text)
+        {
+            var dontWait = new MessageDialog(text);
```

Comment:
>dontWait [](start = 16, length = 8)

Variable name: this could be better now that we are moving hte old code around :) - how about escMessageDialog or just even messageDialog?